File: app/src/main/java/com/softhink/single/survey/SurveyActivity.java
package com.softhink.single.survey;

import android.support.v4.app.FragmentManager;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import com.softhink.single.BaseActivity;
import com.softhink.single.Constants;
import com.softhink.single.R;

public class SurveyActivity extends BaseActivity implements View.OnClickListener{

    private Button btnStartSurvey;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_preferences);

        setUpToolbar("Encuesta", false);

        btnStartSurvey = findViewById(R.id.btnSurvey);
        btnStartSurvey.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {

        switch (v.getId()){
            case R.id.btnSurvey:
                setNavigationUp();
                fragmentManager = getSupportFragmentManager();
                fragmentManager.beginTransaction().add(R.id.containerPreferences, new PreferencesFragment(),
                        Constants.PREFERENCESFRAGMENT).commit();
                break;
        }

    }
}

File: app/src/main/java/com/softhink/single/survey/SurveyActivity.kt
package com.softhink.single.survey

import android.os.Bundle
import android.view.View
import com.softhink.single.BaseActivity
import com.softhink.single.Constants
import com.softhink.single.R
import kotlinx.android.synthetic.main.survey_layout.*

class SurveyActivity : BaseActivity(), View.OnClickListener {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_preferences)

        setUpToolbar("Encuesta", false)

        btnSurvey.setOnClickListener(this)
    }

    override fun onClick(v: View) {
        when (v.id) {
            R.id.btnSurvey -> {
                setNavigationUp()
                mFragmentManager.beginTransaction()
                        .add(R.id.containerPreferences, PreferencesFragment(),
                                Constants.PREFERENCESFRAGMENT).commit()
            }
        }

    }
}
